integr8ly/integreatly-operator
# MGDAPI-409 Added alert for multi-az pod distribution and link to SOP

# Description
Added alert for multi-AZ pod distribution and SOP link

To verify, 
1. Install RHOAM on a multi-az cluster. 
2. when cluster is installed, update local aws cli credential details with AWS details for the cluster. 
3. Run `./scripts/disableAZ.sh true us-east-1c` to disable the `us-east-1c` zone. Change details if needed depending on where the cluster is installed. 
4. Wait for all pods to be recreated in the remaining zones. This can take some time and the console may go offline for a period. 
5. Once pods are recreated, re-enable the zone you disabled `./scripts/disableAZ.sh false us-east-1c` 
6. Check the status of the nodes under `compute` and when all are ready, check for firing alert. 
7. Follow SOP to confirm redistribution of pods to the newly enabled zone.
